### PR TITLE
Resolving issues

### PR DESCRIPTION
**Pull request by Melissa-Martinez-XL**
## Ticket

* QA Issues

## Type of change

* [X] Fix
* [ ] Story
* [ ] Chore

## Description of the change

I fixed the visual issues.

## Screenshot/Execution

https://github.com/xmartlabs/xl-blog/assets/107500715/61c40296-f9c3-4eee-9976-61290d5e501e

imagen

## Related PRs
Links to the related PRs

### CODE REVIEW

File: src/components/footer/footer.module.scss [hunk 1]
```diff
@@ -12,8 +12,5 @@
 }
 
 @media screen and (max-width: map-get($map: $screen-breakpoint, $key: sm)) {
```
Comment (lauta2912) [inline src/components/footer/footer.module.scss:14]: let's remove all the media query block

File: src/components/more-blogs-section/more-blogs-section.jsx [hunk 2]
```diff
@@ -9,14 +9,25 @@ import { Card } from "../card";
 
 import * as styles from './more-blogs-section.module.scss';
 
-const MoreBlogsSection = ({ relatedPosts, refMoreFrom, title, category }) => {
+const MoreBlogsSection = ({ relatedPosts, refMoreFrom, title }) => {
+  const filters = [
```
Comment (lauta2912) [inline src/components/more-blogs-section/more-blogs-section.jsx:13]: This got duplicated, right? why not define it just once and export it?

File: src/components/footer/footer.module.scss (hunk 1 above)
Comment (Melissa-Martinez-XL) [reply src/components/footer/footer.module.scss:14]: Done

File: src/components/more-blogs-section/more-blogs-section.jsx (hunk 2 above)
Comment (Melissa-Martinez-XL) [reply src/components/more-blogs-section/more-blogs-section.jsx:13]: Done

**Review by lauta2912 — APPROVED** [review]

**Pull request merged**